include/ncengine/physics/RigidBody.h

```diff
@@ -182,6 +186,8 @@ class RigidBody
         auto ScalesWithTransform() const -> bool { return !IgnoreTransformScaling(); }
         auto IgnoreTransformScaling() const -> bool { return m_info.flags & RigidBodyFlags::IgnoreTransformScaling; }
         void IgnoreTransformScaling(bool value);
+        auto DisableSleeping() const -> bool { return m_info.flags & RigidBodyFlags::DisableSleeping; }
```

Comment:
We keep weaving in and out of how we should name our getters and setters. I am back on the "GetDisableSleeping" "SetDisableSleeping" train. (No need to change, just think we should converge and then stick there)